copier.yml

```diff
@@ -147,11 +136,11 @@ services_repo:
 
 services_release:
   type: str
-  help: The initial release of the services repository to track e.g. "2024.1.1"
+  help: The initial release of the services repository to track e.g. "2024.12.1" or "main"
```

Comment:
main is not a release - maybe rephrase "release" -> git ref" to make it clear what is expected?

Reply:
I changed it to 'tag or branch'